### DISCUSSION TITLE
Need help with nopk tag

### DISCUSSION THREAD

**Discussion by stasvo** [uptrace/bun#437] [Q&A] [answered]
When I try to create table by use this model I have warning: Wall.ID has unknown tag option: "nopk". I cant understand why its happen. Looks I wrote nopk tag in same way as written in documentation. What I did wrong?

```
type Wall struct {
	ID   int64  `bun:",nopk"`
	Name string `bun:",notnull"`
}
```

**Comment by vmihailenco** [comment, marked as answer]
You did nothing wrong and code be working as expected. Bun should not print a warning in this case.
